src/q6/prostoch-INMA1731/exam/2019/Juin/Majeure/prostoch-INMA1731-exam-2019-Juin-Majeure.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\hypertitle{Stochastic processes}{6}{INMA}{1731}{2019}{June}{Majeure}
+{Alice Borbáth \and Olivier Leblanc \and Julien Verecken}
+{Pierre-Antoine Absil and Luc Vandendorpe}
+
+Exam during 3h with 3 questions: 2 questions by PAA et 1 question by LV.
+
+\section{PAA 1}
+
+Let $G(z)$ be an LTI system which is stable and causal. 
+One inputs a white signal $X$ of variance $\sigma_x^2$ in the system and outputs $Y$.
+The transfer function is given by :
+\begin{equation}
+    G(z)=\frac{z}{\left(z-\frac{1-\epsilon}{\sqrt{2}}(1+j)\right)\left(z-\frac{1-\epsilon}{\sqrt{2}}(1-j)\right)}
+\end{equation}
+\indent where $0<\epsilon\ll1$. Let the input X be white noise with variance $\sigma_X^2$.
+\begin{enumerate}
+    \item Compute the power spectral density of the output $Y$, i.e. $\gamma_Y(z)$.
+    \item Sketch $\gamma_Y(e^{j\Omega})$. In other words, show how the graph of the function $\Omega \mapsto \gamma_Y(e^{j\Omega})$ looks like. Give an approximation of its local extrema (position and value).
+    \item Realize the system by a recurrence equation.
```

Comment:
```suggestion
    \item Define the system by a recurrence equation.
```

I'm not too sure about the wording (and this isn't the exam I took so I can't fall back to what the prof wrote) but "realise" seems like an odd way to word it.